app/src/main/java/com/pysiak/simple/clock/ui/clock/ClockPresenter.java

```diff
@@ -0,0 +1,25 @@
+package com.pysiak.simple.clock.ui.clock;
+
+/**
+ * Presenter for my Alarm fragment with {@link ClockView}
```

Comment:
Alarm? When you'll start to read, check and review what you have done before pushing it to PullRequest?!